ci/test-flow-visibility.sh

```diff
@@ -0,0 +1,156 @@
+    echo "=== antreaflowvisibility-grafana-sankey-plugin installed correctly ==="
+  else
+    echo "=== antreaflowvisibility-grafana-sankey-plugin is NOT installed correctly ==="
+    FAILURE=true
+  fi
+  if ( echo ${grafana_logs} | grep -q 'msg="Plugin registered" logger=plugin.manager pluginId=grafana-clickhouse-datasource' ); then
+    echo "=== grafana-clickhouse-datasource installed correctly ==="
+  else
+    echo "=== grafana-clickhouse-datasource is NOT installed correctly ==="
+    FAILURE=true
+  fi
+}
+
+check_clickhouse_monitor() {
+  echo "=== Check the status of ClickHouse monitor cron job status==="
+  sleep 60
+  CLICKHOUSE_MONITOR=$(kubectl get job -l app=clickhouse-monitor -n flow-visibility -o jsonpath="{.items[-1].metadata.name}")
+  kubectl wait --for=condition=complete job/$CLICKHOUSE_MONITOR -n flow-visibility --timeout=60s
+}
+
+config_antrea() {
+  echo "=== Stopping Antrea === "
+  kubectl delete -f ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+  echo "=== Enable Antrea Flow Exporter ==="
+  sed -i -e "s/#  FlowExporter: false/  FlowExporter: true/g" ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+}
+
+setup_flow_aggregator() {
+  echo "=== Config Antrea Flow Aggregator ==="
+  sed -i -e "s/    #clickHouse:/    clickHouse:/g" ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
+  sed -i -e "s|      #dbUrl: \"tcp://clickhouse-clickhouse.flow-visibility.svc:9000\"|      dbUrl: \"${CLICKHOUSE_URL}\"|g" ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
+  echo "=== Start Antrea Flow Aggregator ==="
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
+  echo "=== Waiting for Antrea Flow Aggregator to be ready ==="
+  kubectl wait --for=condition=ready pod -l app=flow-aggregator -n flow-aggregator --timeout=600s
+}
+
+# Antrea Flow Aggregator starts to insert flow records into ClickHouse database.
+# This function will check if there are flow records inserted in desired table of database.
+check_record() {
+  echo "=== Wait for up to 5 minutes to receive flow records ==="
+  for i in `seq 5`
+  do
+    sleep 60
+    echo "=== Get flow record (try for 1m) ==="
+    clickhouse_output=$(kubectl exec -it chi-clickhouse-clickhouse-0-0-0  -n flow-visibility -- clickhouse-client -h clickhouse-clickhouse.flow-visibility.svc -u clickhouse_operator --password clickhouse_operator_password --query="select count(*) from flows")
+    # Remove all new line, return, tab from the output string, to allow integer comparison.
+    row_number="${clickhouse_output//[$'\t\r\n ']}"
+    echo $row_number
+    if [ $row_number -gt 0 ]; then
+      echo "=== Record is received correctly ==="
+      break
+    elif [ $i == 5 ]; then
+      echo "=== Record is NOT received correctly ==="
+      FAILURE=true
+    fi
+  done
+}
+
+start_antrea
+setup_flow_visibility
+check_grafana
+check_clickhouse_monitor
+config_antrea
+start_antrea
```

Comment:
Dumb question. Why do we need to start at antrea first at line 144?

Reply:
Thanks Salvatore for the reviews. 
We start Antrea first since deploying Flow Visibility stuffs including Grafana, Clickhouse service needs the basic K8s networking provided by Antrea.

Reply:
good point, I was naively thinking it would run in hostnetwork mode!

This however highlights a sort of circular dependency - clickhouse needs antrea becuase it needs a CNI, but the antrea flow collector needs Clickhouse!

However, this does not need to be addressed in this PR, so revolving this comment